docs/family-pub-key.rst

```diff
@@ -0,0 +1,101 @@
+******************************
+Public Key Transaction Family
+******************************
+
+Overview
+========
+
+The *Public Key* transaction family provides a trustless way to store the public key based on RSA cryptosystem on the REMchain.
+
+Definitions of Public Key Entries
+---------------------------------
+
+The following protocol buffers definition define public key entries:
+
+.. code-block:: protobuf
+
+    message PubKeyStorage {
+         // The UID field of a certificate.
+         string owner = 1;
+         // indicates if this key was revoked
```

Comment:
I am pretty sure this string is not in its place :)

Reply:
When copy-paste without even reading 🤦